sagemaker-debugger/amazon_sagemaker_debugger_tutorial_notebooks/debugger_tutorial_built_in_rules_debugging.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
<p>How do we know that the tensors become available in the S3 buckets in 10 minutes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I'll add some more description for this. It takes ~5 mins for initiating an instance and another few mins for the training job to start retrieving tensors, save to the S3 bucket, and retrieve tensors. I suggested 10 minutes to give enough time to see tensors

Reply:
Incorporated

Reply:
I feel that this statement might give the impression that debugger takes ~ 10 minutes, but in this particular case, it is the actual sagemaker training job including (container init) that is contributing to the time duration.

Reply:
I simply removed "10 minutes". Now the sentence is:

> 
To access the saved tensors by Debugger, use the `smdebug` client library to create a Debugger trial object. The following code cell sets up a `tutorial_trial` object, and waits until it finds available tensors from the default S3 bucket.

I think this statement is actually enough, since the trial returns automatic messages until it finds available tensor.